### ISSUE TITLE
larger learning rate + large weight decay performs better?

### ISSUE THREAD

**Issue by askerlee** [lessw2020/Ranger-Deep-Learning-Optimizer#18]
Hi all,
    My colleague and I tried a combination of (relatively) large Ranger learning rate (say, 0.001) + large weight decay (say, 0.1). Seems the large decay leads to better performance? We tried two different models, but both models were customized models, and not standard ones like Resnet.
    Not sure whether anyone else finds similar results.
